doc/api/index.md

```diff
@@ -1391,6 +1411,17 @@ Each of the objects in the returned array have the following properties:
     If not set or set to undefined we don't know whether that video track
     contains an interpretation in sign language.
 
+  - ``isTrickModeTrack`` (``Boolean``): If set to `true`, the track is
+    a trick mode track. The trick mode track proposes video content that is
+    often encoded with a very low framerate because the content is not intended
+    to be played at regular framerate and because the chunks must be faster to
+    load for sthe client.
```

Comment:
I find it risky in terms of API usage and breaking change to just communicate trickmode tracks as well here and in `getVideoTrack`.

At most, we could link the trickmode track to the corresponding video track(s) here by making the former a property of the latter, and maybe add an `active` boolean in it to signal which trick mode track is selected if multiple ones exist

Reply:
Still the case no? We should not have trickMode tracks returned by `getAvailableVideoTracks`

Reply:
But we may have for `getVideoTrack`